frameworks/digital-outcomes-and-specialists-4/questions/briefs/culturalFitCriteriaOutcomes.yml

```diff
@@ -55,7 +55,7 @@ validations:
     message: 'Each requirement must be 30 words or fewer.'
   -
     name: max_items_limit
-    message: 'You must have 20 or fewer criteria.'
+    message: 'You can add up to 20 criteria.'
```

Comment:
It's not possible to show this message - check in code whether it's used somewhere, otherwise delete.

Reply:
Going to leave these in place for now in case we revisit when rebuilding the list-entry component.